scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  appointment_date DATE,
```

Comment:
There won't be a conflict since we're using it as a column name and not an identifier name.